src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-card-color;
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+
+    &:hover {
+      color: $active-title-color;
```

Comment:
Text color should be changes when card is hovered(not only text)

<img width="777" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/4f853699-cbf7-404a-8d51-f599aa79ae14">

Reply:
Also remember about transition